tasks/main.yml

```diff
@@ -1,9 +1,3 @@
 - include: provision.yml
   when: mr_provisioner_do_provision
   delegate_to: localhost
-
-- name: Wait for host for 3600 seconds, but only start checking after 60 seconds
```

Comment:
Why removing it?

Reply:
Because this precise task needs to be called on the newly created host, which did not exist previously. The initial playbook is called with hosts: localhost, then the 'target' host is dynamically created (not written in the host file), and then the wait_for_connection is called on this 'target' host.
I can put a playbook example on pastebin if need be.